Sources/Views/Badges/BadgeView.swift

```diff
@@ -79,14 +88,19 @@ open class BadgeView: BadgeContainerView {
     private func setup() {
         addSubview(imageView)
         imageView.translatesAutoresizingMaskIntoConstraints = false
+        setupImageWidthConstraint()
         NSLayoutConstraint.activate([
             imageView.centerXAnchor.constraint(equalTo: centerXAnchor),
             imageView.centerYAnchor.constraint(equalTo: centerYAnchor),
-            imageView.widthAnchor.constraint(
-                equalTo: widthAnchor, multiplier:
-                Constants.imageViewWidthScale
-            ),
             imageView.heightAnchor.constraint(equalTo: imageView.widthAnchor)
         ])
     }
+
+    private func setupImageWidthConstraint() {
```

Comment:
Let's go for `updateImageWidthConstraint ` as this happens both at `setup` (which is in `init`) and in a `didSet`.